lib/OpenQA/Resource/Locks.pm

```diff
@@ -125,15 +128,24 @@ sub barrier_create {
 }
 
 sub barrier_wait {
-    my ($name, $jobid, $where) = @_;
+    my ($name, $jobid, $where, $check_dead_job) = @_;
     return -1 unless $name && $jobid;
     my $barrier = _get_lock($name, $jobid, $where);
     return -1 unless $barrier;
+    my $jobschema = OpenQA::ResourceAllocator->instance->schema()->resultset("Jobs");
     my @jobs = split(/,/, $barrier->locked_by // '');
+
+    do { $barrier->delete; return -1 }
+      if $check_dead_job
+      && grep { $final_states{$_} }
```

Comment:
Just a cosmetic change, I'd add a line here.

Reply:
You mean a comment line? or..?

Reply:
a blank line between 140 and 141, I just get a bit lost on where the condition ends... :)

Reply:
done! 👍